### DISCUSSION TITLE
Non SPFx

### DISCUSSION THREAD

**Discussion by robpearmain** [n8design/htwoo#32] [Q&A]
Could htwoo be used without SharePoint, for example building a client app in react, and node for the api, completely outside of office 365 etc

**Comment by robpearmain** [comment]
Just wondering if there is an update

**Reply by StfBauer** [reply]
Sorry missed the message completely. @robpearmain yes you can use everything without SharePoint. This is how we use it on the documentation website.

https://lab.n8d.studio/htwoo/htwoo-core/ - There is nothing in their SharePoint related.
